src/config.ts

```diff
@@ -33,6 +34,20 @@ export const WHITELISTED_TOKENS: string[] | '*' = _.isEmpty(process.env.WHITELIS
     ? TokenMetadatasForChains.map(tm => tm.tokenAddresses[CHAIN_ID])
     : assertEnvVarType('WHITELIST_ALL_TOKENS', process.env.WHITELIST_ALL_TOKENS, EnvVarType.WhitelistAllTokens);
 
+// Ignored addresses. These are ignored at the ingress level and are never stored.
+export const IGNORED_ADDRESSES: string[] = _.isEmpty(process.env.IGNORED_ADDRESSES)
+    ? []
+    : assertEnvVarType('IGNORED_ADDRESSES', process.env.IGNORED_ADDRESSES, EnvVarType.AddressList);
+
+// Ignored addresses only for Swap Quote (still present in database and SRA).
+export const SWAP_QUOTE_IGNORED_ADDRESSES: string[] = _.isEmpty(process.env.SWAP_QUOTE_HIDDEN_ADDRESSES)
```

Comment:
nit: this can be just be `swap` as it affects the prices endpoint too I imagine.